kegtab/src/main/res/layout/pour_in_progress_activity.xml

```diff
@@ -38,7 +38,7 @@
 
     <!-- Right box: Camera Preview & Controls -->
 
-    <LinearLayout
+    <RelativeLayout
```

Comment:
This change hide the `Done Pouring` button.

Reply:
Aligned camera view to bottom but reverted changes to Done Button/Image/Shout text to maintain visibility on smaller tablet screens.